llvm/lib/CodeGen/TargetLoweringObjectFileImpl.cpp

```diff
@@ -1688,6 +1688,13 @@ static const Comdat *getWasmComdat(const GlobalValue *GV) {
 }
 
 static SectionKind getWasmKindForNamedSection(StringRef Name, SectionKind K) {
+  // segments within the data section.
```

Comment:
Hmm, somehow you chopped the first line of this comment, but it's there on #49...

Reply:
Oh oops, this didn't merge cleanly in both cases and it looks like I botched this merged but didn't botch the other. Should be fixed now!